File: flowcontrols/DecisionMaking/largest_among_three.py
number1=int(input("Enter number 1"))
number2=int(input("Enter number 2"))
number3=int(input("Enter number 3"))
if number1.__gt__(number2):
    temp = number1
    f=0
else:
    temp = number2
    f=1
if temp.__gt__(number3):
    print(temp,  "is the greatest")
    if f == 0:
        if number2.__gt__(number3):
            print(number2, "is second largest")
        else:
            print(number3, "is second largest")
    else:
        if number1.__gt__(number3):
            print(number1, "is second largest")
        else:
            print(number3, "is second largest")

else:
    print(number3, "is the greatest")
    print(temp, "is second largest")

File: javascript_programs/largest_among_three.js
num1=10
num2=20
num3=5
temp=0
flag=0
if(num1>num2){
    temp=num1
}
else{
    temp=num2
}
if(num3>temp){
    console.log("largest no: "+num3)
    console.log("second largest: "+temp)
}
else{
    console.log("largest no: "+temp)
    if(temp==num1){
        if(num3>num2){
            console.log("second largest: "+num3)
        }
        else{
            console.log("second largest: "+num2)
        }
    }
    else{
        if(num3>num1){
            console.log("second largest: "+num3)
        }
        else{
            console.log("second largest: "+num1)
        }
    }
}
